manifests/db/local.yml

```diff
@@ -77,11 +64,23 @@ instance_groups:
             - username: shield
               password: (( vault meta.vault "/shield:password" ))
               admin: true
+            databases:
+              - name:        (( grab params.ccdb_name ))
+                users:      [(( grab params.ccdb_user ))]
+                extensions: [citext, pgcrypto]
 
-          postgres:
+              - name:        (( grab params.uaadb_name ))
+                users:      [(( grab params.uaadb_user ))]
+                extensions: [citext, pgcrypto]
+
+              - name:        (( grab params.diegodb_name ))
+                users:      [(( grab params.diegodb_user ))]
+                extensions: [citext, pgcrypto]
+                
             config:
               port: 5432
               listen_addresses: '*'
+              max_connections: (( grab params.postgres_max_connections || 100 ))
```

Comment:
Did we document the new max_connections parameter in MANUAL.md?

Reply:
Yep, see `MANUAL.md` L256.